.github/workflows/build-packages.yml

```
@@ -27,115 +27,90 @@ jobs:
-        with:
-          fetch-depth: 0
-          lfs: true
-      - name: Login to DockerHub
-        uses: docker/login-action@v3
-        with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
       - name: Build Documentation
+        run: cd /app && make _build-documentation
         env:
           CESIUM_TOKEN: ${{ secrets.CESIUM_TOKEN }}
```

actionlint (1.7.12)

[error] property "cesium_token" is not defined in object type {actions_runner_debug: string; actions_step_debug: string; dockerhub_token: string; dockerhub_username: string; github_token: string}

(expression)